client/tailscale/localclient_aliases.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package tailscale
+
+import (
+	"context"
+	"crypto/tls"
+
+	"tailscale.com/client/tailscale/apitype"
+	localclient "tailscale.com/localclient/tailscale"
+)
+
+// ErrPeerNotFound is an alias for tailscale.com/localclient/tailscale.
+//
+// Deprecated: import tailscale.com/localclient/tailscale instead.
+var ErrPeerNotFound = localclient.ErrPeerNotFound
+
+// LocalClient is an alias for tailscale.com/localclient/tailscale.
+//
+// Deprecated: import tailscale.com/localclient/tailscale instead.
+type LocalClient = localclient.LocalClient
```

Comment:
maybe the package the should be called "localapi" and the type "Client" so it's `localapi.Client` and `localapi.IPNBusWatch` etc

Reply:
I can see the value in that. I liked keeping it as `tailscale` because that minimizes code changes for consumers (who simply need to update their import path).

Reply:
The import update isn't too bad, even if we do change the name, though: 

```go
// Replace
import "tailscale.com/client/tailscale"

// With
import tailscale "tailscale.com/localapi"
```

I think otherwise nothing should have to change, though, right?

Reply:
Yeah, if we just change the import path without changing the type, we can alias the import as you show. However, if we don't alias the import, then:

1. We stutter, i.e. `localapi.LocalClient`
2. In codebases outside of our own, `localapi` is somewhat ambiguous. Whose local API? Part of why we wanted to name the control API with an import path ending in `tailscale` is that the default package name `tailscale` would provide clarity inside of 3rd party codebases. I think the same is true for the local API, and I think in practice, it should be rare for the same code to import both(?)

Reply:
Having two packages name "tailscale" isn't ideal: one for the web API and for the local API. If your program uses both (as one customer I know does) then you have to rename it on import.

Reply:
So then should either package be named "tailscale", or should they be something like "localclient" and "webclient" (or "apiclient") or something?

Reply:
`local.Client` and `remote.Client` seem reasonable to me. I agree we will want an alias as you noted, at least temporarily, so that existing users don't have to rewrite all their usage sites right away.